### ISSUE TITLE
Could you release your pre-computed results?

### ISSUE THREAD

**Issue by workforai** [HYOJINPARK/Reuse_VOS#5]
Thanks for your work! Could you release your pre-computed results on DAVIS and YouTube-VOS?

**Comment by HYOJINPARK** [comment]
Hi @workforai,
Thanks for your question.

Could you explain more what is pre-computed results??
Do you mean the resulting video?

**Comment by workforai** [comment]
Yes, the predicted masks of DAVIS and YouTube VOS.

**Comment by HYOJINPARK** [comment]
@workforai 
I will go to tomorrow my lab, I will find pre-computed results.
Also, you can use the uploaded 'pth' files to get the result.

**Comment by HYOJINPARK** [comment]
Hi @workforai, 
Thanks for waiting.
The below is my results with ResNet18.
I found there is the same YTB result that I used in the paper, but it looks DAVIS is deleted.
I made it again for DAVIS.
Notice that the parameter of online learning is started from random initialization.
Therefore, the accuracy is slightly different in every try for inference.

YTB2018 with ResNet18
https://drive.google.com/file/d/1sUTgD2YFM9vXqA7iVVJ3uJY1knOBF1rW/view?usp=sharing
DAVIS with ResNet18
https://drive.google.com/file/d/1flqsuX4DbfC3cyMrGN40Jac0KMv9vgDy/view?usp=sharing
